run_tests.sh

```diff
@@ -404,6 +405,7 @@ do
           test_script="pytest"
           report_file="run_framework_tests.html"
           framework_test=1;
+          generate_cwl_conformance_tests=1
```

Comment:
Is it actually needed for framework tests?

Reply:
Since we don't add these to the samples_tool_conf.xml file I don't think they are

Reply:
We do add some of them, the ones used in API tests: https://github.com/galaxyproject/galaxy/pull/12909/files#diff-a4c6a629466aed28ef86982600acabb5642b9c7247318530cea6fd2cc2baabb5

Reply:
Ah, ok, then we need at least those tools.